storage/wdpost_run.go

```diff
@@ -410,6 +441,12 @@ func (s *WindowPoStScheduler) checkNextFaults(ctx context.Context, dlIdx uint64,
 	return faults, sm, nil
 }
 
+// runPost runs a full cycle of the PoSt process:
+//
+//  1. performs recovery declarations for the next deadline.
+//  2. performs fault declarations for the next deadline.
+//  3. computes and submits proofs, batching partitions and making sure they
+//     don't exceed message capacity.
 func (s *WindowPoStScheduler) runPost(ctx context.Context, di dline.Info, ts *types.TipSet) ([]miner.SubmitWindowedPoStParams, error) {
```

Comment:
Nit: Similar to my comment from above, wouldn't it be better if this func is called `runPoSt` to make it clear that this is proof of space time?

Reply:
Yep, let's rename it with the right capitalisation.